pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -612,7 +612,7 @@ func NewMigrationPolicyCrd() (*extv1.CustomResourceDefinition, error) {
 }
 
 // Used by manifest generation
-func NewKubeVirtCR(namespace string, pullPolicy corev1.PullPolicy, featureGates string) *virtv1.KubeVirt {
+func NewKubeVirtCR(namespace string, pullPolicy corev1.PullPolicy, featureGates string, infraReplicas uint8) *virtv1.KubeVirt {
```

Comment:
just curious, why `uint8`? :)

Reply:
That's what `replicas` takes. When creating the `replicas` option, I thought having more than 255 replicas would be unrealistic, and I have old habits dating from back when memory mattered :)

Reply:
:)